### PR TITLE
[NETBEANS-870] Upgrade PostgreSQL driver to version 42.2.5.

### PR DESCRIPTION
**Pull request by Cenbe**
Modify files used to download driver from Maven.

### CODE REVIEW

File: ide/db.drivers/src/org/netbeans/modules/db/drivers/PostgreSQLDriver.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
         <!-- please check with mkleint@netbeans.org before/after updating this or "classpath" section -->
         <property>
             <name>maven-dependencies</name>
-            <value>org.postgresql:postgresql:9.4.1209:jar</value>
+            <value>org.postgresql:postgresql-42.2.5.jar:jar</value>
```
Comment (matthiasblaesing) [inline ide/db.drivers/src/org/netbeans/modules/db/drivers/PostgreSQLDriver.xml]: This should be: org.postgresql:postgresql:42.2.5:jar (the ".jar" in the middle does not belong there). This are maven coordinates in the compact form.

Comment (Cenbe) [reply ide/db.drivers/src/org/netbeans/modules/db/drivers/PostgreSQLDriver.xml]: Fixed in 4e44c52268902c1213110ad0eb33863c2cba7e0f.

File: ide/db.drivers/src/org/netbeans/modules/db/drivers/PostgreSQLDriver.xml [hunk 2]
```diff
@@ -39,7 +39,7 @@
         <!-- please check with mkleint@netbeans.org before/after updating this or "classpath" section -->
         <property>
             <name>maven-dependencies</name>
-            <value>org.postgresql:postgresql:9.4.1209:jar</value>
+            <value>org.postgresql:postgresql-42.2.5:jar</value>
```
Comment (matthiasblaesing) [inline ide/db.drivers/src/org/netbeans/modules/db/drivers/PostgreSQLDriver.xml]: Sorry - I overlooked the hyphen: `org.postgresql:postgresql:42.2.5:jar`

**Comment by matthiasblaesing** [comment]
Please note: I'll update the PR with the corrected maven coordinates. We don't need to play commit ping-pong for a single char change. I'm currently doing a test build locally, if that succeeds, I'll push the squashed commits and will mark that commit, that this PR should be closed. You will stay the author in the final squashed commit.

Thank you!

**Pull request closed without merging**